qiskit/primitives/base/base_sampler.py

```diff
@@ -200,3 +258,33 @@ def parameters(self) -> tuple[ParameterView, ...]:
             List of the parameters in each quantum circuit.
         """
         return tuple(self._parameters)
+
+
+BaseSampler = BaseSamplerV1
+
+
+class BaseSamplerV2:
+    """Sampler base class version 2.
+
+    A Sampler returns samples of bitstrings of quantum circuits.
```

Comment:
More dynamic circuit friendly language

```suggestion
    A Sampler returns samples of quantum circuit outputs.
```